database/schema.sql

```diff
@@ -0,0 +1,42 @@
+DROP TABLE IF EXISTS users;
+CREATE TABLE users (
+  user_id SERIAL PRIMARY KEY,
+  email VARCHAR(255),
+  password VARCHAR(255)
+);
+
+DROP TABLE IF EXISTS projects;
+CREATE TABLE projects (
+  project_id SERIAL PRIMARY KEY,
+  project_name VARCHAR(255) NOT NULL,
+  project_description VARCHAR(255),
+  completed BOOLEAN DEFAULT false,
+  project_rank INTEGER UNIQUE
```

Comment:
I don't think you can say `UNIQUE` here. I think you need to make a unique index.